src/dmd/dtoh.d

```diff
@@ -422,11 +422,7 @@ public:
 
         if (isBuildingCompiler && s.getModule() && s.getModule().isFrontendModule())
         {
-            if (printIgnored)
-            {
-                buf.printf("// ignored %s %s", s.kind(), s.toPrettyChars());
-                buf.writenl();
-            }
+            ignore("// ignored %s %s", s.kind(), s.toPrettyChars());
```

Comment:
if you're refactoring you may as well refactor the leading `"// ignored"`

Reply:
Good idea, done.